salt/states/boto_iam.py

```diff
@@ -472,16 +476,32 @@ def group_present(name, policies=None, policies_from_pillars=None, users=None, r
     if not _ret['result']:
         ret['result'] = _ret['result']
         return ret
-    # TODO: make this work like _group_policies_present. This is currently not
-    #       fully idempotent.
     if users:
         log.debug('Users are : {0}.'.format(users))
         for user in users:
             log.debug('User is : {0}.'.format(user))
             ret = _case_group(ret, user, name, region, key, keyid, profile)
+        for user in _case_delete(users, exists):
```

Comment:
I'm using _case_group in user_present as well. Thats why I didn't add it there. I can change the function name and comment if you agree.